src/Gaufrette/Adapter/AwsS3.php

```diff
@@ -110,8 +110,8 @@ public function rename($sourceKey, $targetKey)
         );
 
         try {
-            $this->service->copyObject($options);
-            return true;
+            $this->service->copyObject(array_merge($options, $this->getMetadata($targetKey)));
```

Comment:
I don't think it's necessary to call `getMetadata` here because [`getOptions`](https://github.com/KnpLabs/Gaufrette/blob/master/src/Gaufrette/Adapter/AwsS3.php#L275) already call `getMetadata`.